packages/dds/sequence/src/test/fuzz/fuzzUtils.ts

```diff
@@ -339,9 +346,18 @@ export function createSharedStringGeneratorOperations(
 	}
 
 	async function obliterateRange(state: ClientOpState): Promise<ObliterateRange> {
+		const max = state.client.channel.getLength() - 1;
+		const num1 = state.random.integer(0, max);
+		const num2 = state.random.integer(0, max);
+		const start = Math.min(num1, num2);
+		const end = Math.max(num1, num2);
+		const startSide =
+			start === end ? Side.Before : state.random.pick([Side.Before, Side.After]);
+		const endSide = start === end ? Side.After : state.random.pick([Side.Before, Side.After]);
 		return {
 			type: "obliterateRange",
-			...exclusiveRange(state),
+			start: { pos: start, side: startSide },
```

Comment:
I also don't think we test obliterate with number endpoints in fuzz - we should probably allow that as well.

Reply:
Are the codepaths any different from when both sides are Before?

Reply:
They end up going through the same code paths eventually (we normalize the numerical obliterate to an equivalent sided one), but it seems fine to test here that nothing is weird about that process. Done.